monitor/pom.xml

```diff
@@ -132,6 +132,12 @@
 					</exclusion>
 				</exclusions>
 			</dependency>
+			<!-- spring-security-cas imports version 3.1, while hadoop-common imports version 3.2.1. -->
+			<dependency>
+				<groupId>commons-collections</groupId>
+				<artifactId>commons-collections</artifactId>
+				<version>3.2.1</version>
```

Comment:
What I mean is that commons-collections is already a transitive dependency of both desktop and monitor:

![image](https://cloud.githubusercontent.com/assets/291450/10480485/c535ec7c-726a-11e5-9f33-13707fc2ee77.png)

So if we start doing management of it, it should be at the parent POM level to avoid anomalies between desktop and monitor.